integration/sawtooth_integration/tests/test_shutdown_smoke.py

```diff
@@ -0,0 +1,111 @@
+# Copyright 2017 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ------------------------------------------------------------------------------
+
+import unittest
+
+import logging
+import subprocess
+import time
+
+LOGGER = logging.getLogger(__name__)
+LOGGER.setLevel(logging.DEBUG)
+
+
+class TestShutdownSmoke(unittest.TestCase):
```

Comment:
It might be worth (in a future PR) using the manage library to abstract the implementation specifics so the test itself isn't docker specific.  (This is not a request to update this PR.)